src/from_substrait.cpp

```diff
@@ -174,7 +174,7 @@ Value TransformLiteralToValue(const substrait::Expression_Literal &literal) {
 	case substrait::Expression_Literal::LiteralTypeCase::kVarChar:
 		return {literal.var_char().value()};
 	default:
-		throw SyntaxException("literals of this type number are not implemented: " + to_string(literal.literal_type_case()));
+		throw NotImplementedException("literals of this type number are not implemented: " + to_string(literal.literal_type_case()));
```

Comment:
Should we try to get the field name and use it in the exception message?
`literal.GetDescriptor()->FindFieldByNumber(literal.literal_type_case())->name()`

Reply:
Nice, I've updated everywhere I could find to use this construct.